src/main/java/Randomizer.java

```diff
@@ -0,0 +1,41 @@
+public class Randomizer {
+    private static final int NUMBER_OF_COLOURS = 4;
+    private static final int NUMBER_OF_FIGURES = 4;
+    private static final int MAX_NUMBER_OF_FIGURES = 30;
+    private static final int MAX_VALUE = 10;
+
+    public String choseFigure() {
+        FigureTypes[] figureTypes = FigureTypes.values();
+        return figureTypes[(int) (Math.random()*NUMBER_OF_FIGURES)].figure;
```

Comment:
```suggestion
        return figureTypes[(int) (Math.random() * NUMBER_OF_FIGURES)].figure;
```
don't forget about spaces and codestyle